transformer_engine/pytorch/jit.py

```diff
@@ -35,6 +35,14 @@ def bias_gelu_fused_(inp: torch.Tensor, bias: torch.Tensor) -> torch.Tensor:
     x = inp + bias
     return x * 0.5 * (1.0 + torch.tanh(0.79788456 * x * (1 + 0.044715 * x * x)))
 
+@torch.jit.script
```

Comment:
This needs to be wrapped around `torch.cuda.amp.autocast(enabled=False)` similar to the other fused functions in order to work with AMP. Otherwise the generated kernel will return an fp32 output regardless.

Reply:
both are already wrapped in `torch.cuda.amp.autocast(enabled=False)` in `bias_gelu_fused` and `bgrad_dgelu_fused`

Reply:
I missed that, thanks